contrib/bloop/src/mill.contrib.bloop/BloopModule.scala

```diff
@@ -0,0 +1,197 @@
+package mill.contrib.bloop
+
+import ammonite.ops._
+import bloop.config.Config.{File => BloopFile}
+import bloop.config.ConfigEncoderDecoders._
+import io.circe.Encoder
+import mill._
+import mill.define._
+import mill.eval.Evaluator
+import mill.modules.Jvm
+import mill.scalalib._
+import ujson.Visitor
+import upickle.default
+
+// TODO diamond problem is going to make this tricky. Using a self-type appears
+// tobe problematic with mill. Moreover, the global "install"
+// method that inspects the build relies on whether the modules are instances of
+// JavaModule or ScalaModule, so not quite sure what approach to take to avoid
+// copy pasting between a "BloopJavaModule" and a "BloopScalaModule" (and a potential
+// BloopScalaJsModule ?)
+//
+// TODO : bloop clean doesn't seem to work for me. Also, investigate if it's
```

Comment:
It should be fixed in latest master. You can install it by following our CONTRIBUTING guidelines.

Reply:
Haha I don't know if you remember, but I had had unfortunate experiences with installing bloop outside of brew (files being deleted arbitrarily, not very keen to retry that)

Reply:
Ha! Is it ok if I redirect you to one of the latest binaries in our Bintray repo? :smile:

Reply:
Now that I think about it, iirc the problem was not with the installation process but rather after the second run of a bloop command (my assumption is that it's a nailgun related problem). 

Anyhow, I'll give it a go 👍 (probably in a docker environment just to be safe)

Reply:
I'm pushing in another PR a way to install local versions of bloop with brew, wait until that hits master and it should work :smile: